viewing_party/party.py

```diff
@@ -1,23 +1,160 @@
 # ------------- WAVE 1 --------------------
-
+import copy
+#Creates a dictionary representing a movie with title, genre and rating.
 def create_movie(title, genre, rating):
-    pass
-
+    if title and genre and rating:
+        return {"title": title, "genre": genre, "rating": rating} #it's a movie_dic
+    return None
 # -----------------------------------------
-# ------------- WAVE 2 --------------------
+#Adds a movie to the user's watched list.
+def add_to_watched(user_data, movie): 
+    if "watched" not in user_data:
+        user_data["watched"] = []
+    user_data["watched"].append(movie)
+    return user_data
 # -----------------------------------------
+#Adds a movie to the user's watchils.
+def add_to_watchlist(user_data, movie):
+    if "watchlist" not in user_data:
+        user_data["watchlist"] = []
+    user_data["watchlist"].append(movie)
+    return user_data

```

Comment:
Like my comment above, given that the requirements tell us "the value of user_data will be a dictionary with a key `watchlist`", we can directly append the movie to `watchlist`. 

```suggestion
    user_data["watchlist"].append(movie)
    return user_data

```